2021-05-24 01:30:47,698 INFO    MainThread:2517 [wandb_setup.py:_flush():69] setting env: {}
2021-05-24 01:30:47,698 INFO    MainThread:2517 [wandb_setup.py:_flush():69] setting login settings: {}
2021-05-24 01:30:47,698 INFO    MainThread:2517 [wandb_init.py:_log_setup():336] Logging user logs to /home/ubuntu/data/matengfei/kg-mtl/wandb/run-20210524_013047-3njkynig/logs/debug.log
2021-05-24 01:30:47,698 INFO    MainThread:2517 [wandb_init.py:_log_setup():337] Logging internal logs to /home/ubuntu/data/matengfei/kg-mtl/wandb/run-20210524_013047-3njkynig/logs/debug-internal.log
2021-05-24 01:30:47,698 INFO    MainThread:2517 [wandb_init.py:init():369] calling init triggers
2021-05-24 01:30:47,698 INFO    MainThread:2517 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2021-05-24 01:30:47,699 INFO    MainThread:2517 [wandb_init.py:init():418] starting backend
2021-05-24 01:30:47,699 INFO    MainThread:2517 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-05-24 01:30:47,705 INFO    MainThread:2517 [backend.py:ensure_launched():132] starting backend process...
2021-05-24 01:30:47,709 INFO    MainThread:2517 [backend.py:ensure_launched():137] started backend process with pid: 2529
2021-05-24 01:30:47,711 INFO    MainThread:2517 [wandb_init.py:init():423] backend started and connected
2021-05-24 01:30:47,712 INFO    MainThread:2517 [wandb_init.py:init():465] updated telemetry
2021-05-24 01:30:47,712 INFO    MainThread:2517 [wandb_init.py:init():484] communicating current version
2021-05-24 01:30:48,132 INFO    MainThread:2517 [wandb_init.py:init():489] got version response 
2021-05-24 01:30:48,132 INFO    MainThread:2517 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-05-24 01:30:48,348 INFO    MainThread:2517 [wandb_init.py:init():522] starting run threads in backend
2021-05-24 01:30:50,583 INFO    MainThread:2517 [wandb_run.py:_console_start():1565] atexit reg
2021-05-24 01:30:50,584 INFO    MainThread:2517 [wandb_run.py:_redirect():1412] redirect: SettingsConsole.REDIRECT
2021-05-24 01:30:50,584 INFO    MainThread:2517 [wandb_run.py:_redirect():1417] Redirecting console.
2021-05-24 01:30:50,586 INFO    MainThread:2517 [wandb_run.py:_redirect():1479] Redirects installed.
2021-05-24 01:30:50,586 INFO    MainThread:2517 [wandb_init.py:init():546] run started, returning control to user process
2021-05-24 01:30:50,590 INFO    MainThread:2517 [wandb_run.py:_atexit_cleanup():1535] got exitcode: 2
2021-05-24 01:30:50,590 INFO    MainThread:2517 [wandb_run.py:_restore():1507] restore
2021-05-24 01:30:52,595 INFO    MainThread:2517 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 844
  total_bytes: 844
}

2021-05-24 01:30:52,701 INFO    MainThread:2517 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 844
  total_bytes: 844
}

2021-05-24 01:30:52,886 INFO    MainThread:2517 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 844
  total_bytes: 844
}

2021-05-24 01:30:53,357 INFO    MainThread:2517 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 844
  total_bytes: 5656
}

2021-05-24 01:30:53,675 INFO    MainThread:2517 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7125
  total_bytes: 7125
}

2021-05-24 01:30:53,777 INFO    MainThread:2517 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7125
  total_bytes: 7125
}

2021-05-24 01:30:53,880 INFO    MainThread:2517 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7125
  total_bytes: 7125
}

2021-05-24 01:30:53,982 INFO    MainThread:2517 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7125
  total_bytes: 7125
}

2021-05-24 01:30:54,084 INFO    MainThread:2517 [wandb_run.py:_wait_for_finish():1657] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7125
  total_bytes: 7125
}

2021-05-24 01:30:55,162 INFO    MainThread:2517 [wandb_run.py:_show_files():1879] logging synced files
